src/src/DataStructs/SettingsStruct.cpp

```diff
@@ -158,6 +158,16 @@ void SettingsStruct_tmpl<N_TASKS>::ApDontForceSetup(bool value) {
   bitWrite(VariousBits1, 14, value);
 }
 
+template<unsigned int N_TASKS>
+bool SettingsStruct_tmpl<N_TASKS>::JSONBoolWithQuotes() const {
+  return bitRead(VariousBits1, 15);
+}
+
+template<unsigned int N_TASKS>
+void SettingsStruct_tmpl<N_TASKS>::JSONBoolWithQuotes(bool value) {
+  bitWrite(VariousBits1, 15, value);
```

Comment:
I have a PR pending, can you change this bit index to `16`?

Reply:
😄 I saw the other commit and was thinking that it would be a challenging merge. I'll use 16.